Yoast/javascript
# Add sentence length assessment for Hungarian

## Summary


This PR can be summarized in the following changelog entry:

* [yoastseo] Adds specs for the sentence length assessment in Hungarian.

## Relevant technical choices:

* Research has confirmed that the default value of 20 words is a good value for Hungarian.


## Test instructions

This PR can be tested by following these steps:



* Set your website to Hungarian.  
* Create a new post 
* Add a text with 3 Hungarian sentences with 20 words, e.g.:
     `A magyar nyelv az uráli nyelvcsalád tagja, a finnugor nyelvek közé tartozó ugor nyelvek egyike legközelebbi rokonai a 
      manysi és.`
* Make sure that your sentence length assessment score reads: `Sentence length: Great!`
* Make all of your sentence longer than 20 words.
* Check that the Sentence length assessment score becomes worse, e.g.: `Sentence length: 100% of the sentences contain more than 20 words, which is more than the recommended maximum of 25%. Try to shorten the sentences`
* Make the sentences shorter than 20 words and check that bullet turns green again.

## Impact check

* This PR affects the following parts of the plugin, which may require extra testing:
  *

## UI changes

## Quality assurance


Fixes https://yoast.atlassian.net/browse/LINGO-490
